includes/core-changes/aspnetcore/3.0/targetfx-netfx-tfm-support.md

```diff
@@ -0,0 +1,57 @@
+### Target framework: .NET Framework support dropped
+
+Starting with ASP.NET Core 3.0, .NET Framework is an unsupported target framework.
+
+#### Change description
+
+.NET Framework will get less of the new platform and language features that come to .NET Core. The reasoning is the in-place update nature of .NET Framework and the decision to limit changes there that might break existing apps. To ensure ASP.NET Core can use the improvements coming to .NET Core, ASP.NET Core will only run on .NET Core starting from 3.0. You can think of ASP.NET Core 3.0 as being part of .NET Core.
```

Comment:
shouldn't we be more explicit here? We've already announced that 4.8 was the last major version of .NET Framework:
https://devblogs.microsoft.com/dotnet/net-core-is-the-future-of-net/